gui/src/streamsession.cpp

```diff
@@ -164,6 +164,12 @@ void StreamSession::HandleKeyboardEvent(QKeyEvent *event)
 		case Qt::Key::Key_Backspace:
 			button_mask = CHIAKI_CONTROLLER_BUTTON_MOON;
 			break;
+		case Qt::Key::Key_Backslash:
+			button_mask = CHIAKI_CONTROLLER_BUTTON_BOX;
+			break;
+		case Qt::Key::Key_BracketRight:
```

Comment:
`\` and `]` can be hard to press on some layouts, for me it's AltGr+7 and AltGr+ß for example. How about adding `X` and `C` in addition to `\` and `]`?

Reply:
Done. I picked these because they happened to be right next to enter and backspace on my keyboard. But hey, having more options is probably better for everybody.